README.md

```diff
@@ -101,13 +101,11 @@ If you are hosting a call for the first time or interested in learning how to do
 
 ### Weekly/BiWeekly Syncs of Working Groups, Special Interest Groups, and Related Projects
 
-- [`JS Core Dev`](https://github.com/ipfs/team-mgmt/issues/650)
-- [`Go Core Dev`](https://github.com/ipfs/team-mgmt/issues/674)
-- [`Project WG`](https://github.com/ipfs/project#ipfs-project-working-group)
+- [`Core Implementations`](https://github.com/ipfs/team-mgmt/issues/992)
+- [`Project Operations`](https://github.com/ipfs/project#ipfs-project-working-group)
```

Comment:
Shall we rename the repo as well? I do like the name _project operations_ a lot, way less misleading.

Reply:
```suggestion
- [`Project Operations`](https://github.com/ipfs/project-operations#ipfs-project-working-group)
```